catalog/plugin/catalog-plugin-metacard-validation/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -44,16 +44,16 @@
 
     </bean>
 
-    <!-- Pre-Query Metacard Validity Checker Plugin -->
-    <bean id="pre-query-plugin"
-          class="ddf.catalog.metacard.validation.MetacardValidityCheckerPlugin">
-        <cm:managed-properties
-                persistent-id="ddf.catalog.metacard.validation.MetacardValidityCheckerPlugin"
-                update-strategy="container-managed"/>
-        <argument ref="filterBuilder"/>
-        <argument ref="filterAdapter"/>
-        <property name="showInvalidMetacards" value="false"/>
-    </bean>
+    <!--&lt;!&ndash; Pre-Query Metacard Validity Checker Plugin &ndash;&gt;-->
```

Comment:
No, I commented them out to run it without the services, but also without deleting them (just in case it broke, I could fix it easier). I intend to delete this in the final version, thanks for the reminder :)